spec/controllers/user_registrations_controller_spec.rb

```diff
@@ -21,7 +21,6 @@
       response.status.should == 200
       json = JSON.parse(response.body)
       json.should == {"email" => "test@test.com"}
-      controller.spree_current_user.email.should == "test@test.com"
```

Comment:
Shouldn't we test that `spree_current_user` is `nil` instead? There is still behaviour we need to test.

Reply:
Done.

Reply:
Code hasn't changed?

Reply:
I did it with a rebase and force push. It's not updating it in the comments view, but it's there: https://github.com/Matt-Yorkley/openfoodnetwork/blob/b4f9ef4d4af1bf03e6222abca473fb12ef287932/spec/controllers/user_registrations_controller_spec.rb#L24